pallets/teerex/src/tests/test_cases.rs

```diff
@@ -1026,3 +1026,31 @@ fn publish_hash_with_too_much_data_fails() {
 		);
 	})
 }
+
+#[test]
+fn add_enclave_works_exceeds_number() {
+	new_test_ext().execute_with(|| {
+		crate::EnclaveCountMax::<Test>::put(0u64);
+		assert_eq!(Teerex::enclave_count_max(), 0);
```

Comment:
If this is 0, why the test pass ?
I would expect that no enclaves can be registered.

Reply:
Yes, now no enclaves can be registered.